docs/metadata.rst

```diff
@@ -21,9 +21,11 @@ the :ref:`Standard JSON Interface<compiler-api>`.
 
 You have to publish the metadata file to IPFS, Swarm, or another service so
 that others can access it. You create the file by using the ``solc --metadata``
-command that generates a file called ``ContractName_meta.json``. It contains
+command that generates a JSON to the standart output. It contains
```

Comment:
We actually DO generate that file, but only when the `--output-dir` option is given. We should reflect that in the documentation though.

Reply:
I have modified the sentence.